wallet/src/de/schildbach/wallet/BaseWalletApplication.kt

```diff
@@ -1,67 +0,0 @@
-/*
- * Copyright 2020 Dash Core Group.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-package de.schildbach.wallet
-
-import android.os.Handler
-import android.os.HandlerThread
-import android.os.Process
-import androidx.lifecycle.LiveData
-import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.switchMap
-import androidx.multidex.MultiDexApplication
-import de.schildbach.wallet.rates.ExchangeRatesRepository
-import org.bitcoinj.core.Address
-import org.dash.wallet.common.WalletDataProvider
-import org.dash.wallet.common.data.ExchangeRateData
-
-// TODO: do we need this class?
-abstract class BaseWalletApplication : MultiDexApplication(), WalletDataProvider {
```

Comment:
I don't think we need this class. Seems like it was intended to provide some shared logic to the integration modules; not needed with the current DI scheme.